app/maui-blazor/Services/ServiceImplementations.cs

```diff
@@ -0,0 +1,271 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Threading;
+using CommunityToolkit.Maui.Media;
+
+namespace MauiBlazor.Services;
+
+// NOTE: These service implementations are incomplete. Only a few parts are implemented
+// for basic app functionality (for now).
+
+public class MauiSessionStorageService : ISessionStorageService
+{
+    public double Length => 0;
+
+    public void Clear()
+    {
+    }
+
+    public TValue? GetItem<TValue>(string key, JsonSerializerOptions? options = null)
+    {
+        return default;
+    }
+
+    public string? Key(double index)
+    {
+        return default;
+    }
+
+    public void RemoveItem(string key)
+    {
+    }
+
+    public void SetItem<TValue>(string key, TValue value, JsonSerializerOptions? options = null)
+    {
+    }
+}
```

Comment:
Implement this with a `ConcurrentDictionary<string, object>()`. 

```suggestion
public class MauiSessionStorageService : ISessionStorageService
{
    private readonly ConcurrentDictionary<string, object> _storage = new(StringComparison.OrdinalIgnoreCase);

    public double Length => _storage.Length;

    public void Clear() => _storage.Clear();

    public TValue? GetItem<TValue>(string key, JsonSerializerOptions? options = null)
    {
        // Ignore these...
        _ = options;
        
        return _storage.TryGetValue(key, out var value)
            ? value as TValue
            : default;
    }

    public string? Key(double index) => default;

    public void RemoveItem(string key)
    {
        _storage.TryRemove(key, out var _);
    }

    public void SetItem<TValue>(string key, TValue value, JsonSerializerOptions? options = null)
    {
        // Ignore these...
        _ = options;
        
        _storage.AddOrUpdate(key, value, (oldValue, newValue) => newValue);
    }
}
```